content/article/passing-by.md

```diff
@@ -0,0 +1,237 @@
+{
+   "image" : "/images/passing-by/relay.jpg",
+   "thumbnail" : "/images/passing-by/relay.jpg",
+   "date" : "2021-07-13T11:48:52",
+   "categories" : "development",
+   "title" : "Passing by",
+   "tags" : [
+      "reference",
+      "functions"
+   ],
+   "draft" : false,
+   "description" : "Looking at how arguments are passed to subs",
+   "authors" : [
+      "thibault-duponchelle"
+   ]
+}
+
+## Assignments by copy or reference
+When assigning, `perl` often copies the value, see the following code:
```

Comment:
"often" makes it sound like there are times when this won't happen, aside from the reference / alias issue. You might start with the general case (that perl assigns by value), then discuss the cases where it doesn't.

Reply:
OK :)